### ISSUE TITLE
Add possibility to store output information in file

### ISSUE THREAD

**Issue by vbauer** [fcannizzaro/gradver#2]
Is it possible to add parameter to store result information in some file (XML)?
It will be useful to make external integration with gradver.

**Comment by fcannizzaro** [comment]
Something like this?

```xml
<?xml version="1.0"?>
<gradver>
	<dependencies>
		<dependence>
			<package>com.squareup.okhttp</package>
			<library>okhttp-urlconnection</library>
			<version>2.7.2</version>
			<latest>2.7.4</latest>
		</dependence>
		<dependence>
			<package>com.github.machinarius</package>
			<library>preferencefragment</library>
			<version>0.1.1</version>
			<latest>0.1.2</latest>
		</dependence>
		<dependence>
			<package>com.github.paolorotolo</package>
			<library>appintro</library>
			<version>3.3.0</version>
			<latest>3.4.0</latest>
		</dependence>
		<dependence>
			<package>com.github.afollestad.material-dialogs</package>
			<library>commons</library>
			<version>0.8.5.3</version>
			<latest>0.8.5.4</latest>
		</dependence>
		<dependence>
			<package>com.mikepenz</package>
			<library>materialdrawer</library>
			<version>4.6.4</version>
			<latest>5.0.0</latest>
		</dependence>
		<dependence>
			<package>com.github.Kennyc1012</package>
			<library>BottomSheet</library>
			<version>2.0.7</version>
			<latest>2.1.1</latest>
		</dependence>
		<dependence>
			<package>com.mikepenz</package>
			<library>google-material-typeface</library>
			<version>2.2.0.1</version>
			<latest>2.2.0.1.original</latest>
		</dependence>
	</dependencies>
</gradver>
```

**Comment by vbauer** [comment]
Yes!

**Comment by vbauer** [comment]
Sorry, could you please explain how to use it?
I've tried the following combinations:

```bash
gradver --export
gradver --export=output.xml
gradver --export output.xml
```

but I don't see any output files..

**Comment by fcannizzaro** [comment]
Now available [Release 1.3.1](https://github.com/fcannizzaro/gradver/releases/tag/1.3.1)
